main.css

```diff
@@ -16,31 +16,171 @@ time, mark, audio, video {
 	border: 0;
 	font-size: 100%;
 	font: inherit;
+	font-family: "Gill Sans", "Gill Sans MT", Calibri, sans-serif;
 	vertical-align: baseline;
 	max-width: none;
 }
 
-html, body{
-
+body{
+	/*background: linear-gradient(black, white);*/
+	background: url(".//assets/backgrounds/background_home.png") no-repeat center center fixed; 
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  background-size: cover;
 }
 
 /*********   header   **********/
+header, footer {
+	width: 100%;
+	height: 60px;
+	position: fixed;
+	text-align: center;
+
+}
 header {
+	background: #e0747f;
+}
+
+footer {
+	background: #6db2e0;
+	/*margin-top: 2em;*/
+	clear: both;
+	/*position:absolute;*/
+	bottom:0; /*idk how this works*/
```

Comment:
you've placed everything inside the footer element fixed to the bottom of the screen. this is why it's overlapping the pokemon images.